### DISCUSSION TITLE
Optimized build portal broken

### DISCUSSION THREAD

**Discussion by jadasse** [86Box/86Box#1314] [Q&A] [answered]
Where can I find the latest optimized build? The link on the front page leads to some error page

**Comment by ppgrainbow** [comment, marked as answer]
The optimised builds have been discontinued and removed from Jenkins. You will have to at least use the stable build of 86Box.

The decision to discontinue the optimised build of 86Box was made by the developers and I hope that you understand their decision.

Sorry about that.

**Reply by jadasse** [reply]
why was it discontinued?

**Reply by ppgrainbow** [reply]
The optimised builds were constantly broken, wasting server space, bandwith and compile time.

To some extent, some of the optimised builds were completed in up to a hour. Not a good thing. :|

**Reply by jadasse** [reply]
Ok, thanks!

**Reply by ppgrainbow** [reply]
You're welcome! :)
